app/src/wiki/mw/api/page.js

```
@@ -17,51 +17,51 @@ const defaultThumbnailSize = 120;
-        ...nMap,
-        [normalization.to]: normalization.from,
-      }),
-      {}
-    );
+    const mwApi = siteMapper.getApi(language);
+
+    return mwApi.get(params).then((response) => {
+        const apiResponse = response.query.pages;
+        const redirects = response.query.redirects || [];
+        const redirectMap = redirects.reduce(
+            (rMap, redirect) => ({ ...rMap, [redirect.to]: redirect.from }),
```

Biome (1.9.4)

[error] Avoid the use of spread (`...`) syntax on accumulators.

Spread syntax should be avoided on accumulators (like those in `.reduce`) because it causes a time complexity of `O(n^2)`.
Consider methods such as .splice or .push instead.

(lint/performance/noAccumulatingSpread)